tools/FakeItEasy.Build/Program.cs

```diff
@@ -56,6 +56,19 @@
     forEach: projectsToPack,
     action: project => Run("dotnet", $"pack {project.Path} --configuration Release --no-build --nologo --output \"{Path.GetFullPath("artifacts/output")}\""));
 
+// If mkdocs's site_url configuration is not set, sitemap.xml is full of "None" links.

```

Comment:
our site_url is not set. I think there's some value in setting it, and in theory mike will rewrite it, but didn't want to do here.